cognite_toolkit/_cdf_tk/loaders/_worker.py

```diff
@@ -72,7 +81,7 @@ def prepare_resources(
         is_dry_run: bool = False,
         force_update: bool = False,
         verbose: bool = False,
-    ) -> tuple[T_CogniteResourceList, T_CogniteResourceList, list[T_ID], T_CogniteResourceList]:
+    ) -> CategorizedResources:
```

Comment:
The docstring is wrong now, since it says that a tuple is returned.

Reply:
Updated